kernel/dw1000.c

```diff
@@ -1503,11 +1513,19 @@ static void dw1000_rx_link_qual(struct dw1000 *dw)
 	/* The reported LQI is upper limited */
 	rx->lqi = (snr < DW1000_LQI_MAX) ? snr : DW1000_LQI_MAX;
 
-	/* Check Preamble length */
-	if (rxpacc < psr/4) {
+	/* Check Preamble length - with rather random limit */
```

Comment:
The value 4 has a definition upstream; please rebase, and change the defined constant's value if needed.